src/AccessibilityInsights.SharedUx/Settings/ConfigurationManager.cs

```diff
@@ -29,12 +29,15 @@ public class ConfigurationManager
         /// Event recorder settings
         /// </summary>
         public RecorderSetting EventConfig { get; private set; }
+        
+        public FixedConfigSettingsProvider PathProvider { get; }
```

Comment:
>PathProvider [](start = 43, length = 12)

"PathProvider" as a name seems to imply that we're only providing paths and nothing else